Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/manager_app/view/task_list/TaskListActivity.java

```diff
@@ -68,4 +81,28 @@ private void setUpViews() {
         taskListAdapter = new TaskRecyclerViewAdapter(taskItemClickListener);
         recyclerView.setAdapter(taskListAdapter);
     }
+
+    private void displayTaskCannotBeRemovedAlert(List<PlanTaskTemplate> planTasks) {
+        String planNames = "";
+        for (PlanTaskTemplate planTask : planTasks){
+            String asd = planTask.getPlanTemplate().getName();
+            planNames = planNames + asd + ", ";
+        }
+        planNames = planNames.substring(0, planNames.length() - 2);
+        planNames += ".";
```

Comment:
I think it could be better if we use something like String join utils to get rid of this low level string logic :) for example:
```
List<String> relatedPlansNames = new ArrayList<>();
        for (PlanTaskTemplate planTask : planTasks){
            relatedPlansNames.add(planTask.getPlanTemplate().getName());
        }

        String relatedPlansMessage = TextUtils.join(", " , relatedPlansNames);
```

And then:
```
alertDialog.setMessage(
                getResources().getString(R.string.task_cannot_be_removed_message)
                        + " " + relatedPlansMessage + ".");
```

Reply:
Duh, just noticed the "asd" variable name :d 
Sure, I will look into it.